scheduler/project.clj

```diff
@@ -207,7 +207,9 @@
 
   :main cook.components
   :jvm-opts ["-Dpython.cachedir.skip=true"
-             "-XX:MaxPermSize=500M"
+             "-XX:+UseG1GC"
+             "-XX:+UseStringDeduplication"
+             "-XX:+PrintStringDeduplicationStatistics"
```

Comment:
I had to comment out `-XX:+UseStringDeduplication` and `-XX:+PrintStringDeduplicationStatistics` in order to get my local cook scheduler to start up. Is there something I'm missing @scrosby?